docs/start/hello-world.ipynb

```diff
@@ -7,23 +7,42 @@
    "source": [
     "# Hello world\n",
     "\n",
-    "This Hello world example creates a simple quantum program and runs it on a quantum system. Begin with following the [Install and set up](install) instructions if you haven't already, including the steps to [Set up to use IBM Quantum&trade; Platform](setup-channel#set-up-to-use-ibm-quantum-platform).\n",
+    "This example contains two parts: you will first create a simple quantum program and run it on a quantum system.  Because actual quantum research requires much more robust programs, in the [Scale to large numbers of qubits](#scale-to-large-numbers-of-qubits) section, you will scale the simple program up to utility level.  You can also follow along with the Hello World episode of the Coding with Qiskit 1.0 video series. \n",
```

Comment:
I wonder if we can remove the colon after "two parts" (and maybe replace with a period?) since the part between the colon and the period doesn't explain both parts. Also, there is a reference to "first" but no reference to "second".  Is the second part of the example the scaling up to utility?

Reply:
> I wonder if we can remove the colon after "two parts" (and maybe replace with a period?) since the part between the colon and the period doesn't explain both parts. Also, there is a reference to "first" but no reference to "second". Is the second part of the example the scaling up to utility?

tis